app/controllers/api/v1/accounts_controller.rb

```diff
@@ -45,6 +45,8 @@ def cache_keys
 
   def update
     @account.update!(account_params.slice(:name, :locale, :domain, :support_email, :auto_resolve_duration))
+    @account.custom_attributes.merge!(custom_attributes_params)
+    @account.save!
```

Comment:
avoid multiple saves

```suggestion
    @account.assign_attributes(account_params.slice(:name, :locale, :domain, :support_email, :auto_resolve_duration))
    @account.custom_attributes.merge!(custom_attributes_params)
    @account.save!
```